generator/views/client_counts_view.py

```diff
@@ -67,9 +67,14 @@ class ClientCountsView(View):
         }
     ]
 
-    def __init__(self, namespace: str, tables: List[Dict[str, str]]):
+    def __init__(
+        self,
+        namespace: str,
+        tables: List[Dict[str, str]],
+        name: str = "client_counts",
+    ):
         """Get an instance of a ClientCountsView."""
-        super().__init__(namespace, "client_counts", ClientCountsView.type, tables)
+        super().__init__(namespace, name, ClientCountsView.type, tables)
```

Comment:
this change shouldn't change existing views since all of them are named `client_counts` anyway